tools/integration_tests/tokenserver/test_support.py

```diff
@@ -17,7 +17,6 @@
 
 
 class TestCase:
-    AUTH_METHOD = os.environ.get('TOKENSERVER_AUTH_METHOD', 'oauth')
```

Comment:
This line of code is run a single time when this class definition is read by the interpreter and **not** every time the `TestCase` symbol is referenced (which I think was my original assumption). We need the Tokenserver auth method to be read from the env var every time we run the tests in this class, which is exactly what `setUpClass` is for. I've moved this and the definition of the `_build_auth_headers` function there